crates/config/src/lib.rs

```diff
@@ -356,6 +356,13 @@ pub struct Config {
     ///
     /// This includes what operations can be executed (read, write)
     pub fs_permissions: FsPermissions,
+
+    /// Temporary config to enable [SpecId::CANCUN]
+    ///
+    /// <https://github.com/foundry-rs/foundry/issues/5782>
+    /// Should be removed once EvmVersion Cancun is supported by solc
+    pub cancun: bool,
```

Comment:
wonder if we should prefix this with something like `UNSTABLE`?

Reply:
should be fine, It's only available via config